README.md

```diff
@@ -1,5 +1,345 @@
+      deployed at any time to the live website. When you click the publish
+      button, Storyblok will make a request to Vercel to start a preview
+      deployment.
+   3. Look for deployment notifications in Slack. Vercel will send a message to
+      the `#vercel-notifications` channel containing a link to the preview URL.
+   4. Visit the Vercel preview URL to review and double-check your changes.
+3. Going live
+   1. When you want your changes to go to the live production site, coordinate
+      with the dev team to open a merge pull request to the `main` branch on
+      GitHub.
+   2. When the Vercel GitHub app comments on the merge PR with a preview URL,
+      visit that preview URL to check your changes. (You should also be able to
+      get the Vercel preview URL from the Slack channel.)
+   3. If it all looks good, coordinate with the dev team to merge the PR to
+      `main`.
+   4. Once that PR is merged to main, wait for the production build to finish
+      deploying (check Slack channel for notifications), then check your changes
+      against the live site. You may need to clear your browser's cache before
+      you can see your changes.
+
+### Code changes (GitHub)
+
+Code changes move through three stages, each of which corresponds to a branch in
+git: a feature branch, then the `develop` branch, then `main`. When your code
+gets merged to the main branch, Vercel deploys it the public website.
+
+You should never merge your code into the `develop` branch unless it's ready for
+deployment (via merge to main). Putting your code into the `develop` branch is a
+signal to the rest of your team that says: this code is ready to run on the
+public website.
+
+These are the concrete steps to follow to move your code from branch to branch:
+
+1. Open a pull request against the `develop` branch. This will kick off preview
+   deployments in Vercel. Vercel will add a comment to your pull request with
+   links to the preview URL.
+2. Check one or both preview URLs depending on whether your change affects one
+   or both sites.
+3. Once your pull request has been reviewed and approved, commit it to the
+   `develop` branch.
+4. When you want your code change to go live, open a pull request to merge the
+   `develop` branch into `main`.
+5. Review both preview URLs that Vercel will add to your pull request.
+6. If all looks good and your pull request has gotten approval, then merge it
+   into `main`. This will kick off a production deploy on Vercel. Check the live
+   public websites once the deploy is finished (Vercel will send a notification
+   to the Slack channel). You may need to clear your browser's cache before you
+   can see your changes.
+
+## Integrations
+
+Making changes to the website relies on the ability of several different
+services to interact with each other. This section covers each service one by
+one and how it integrates with some of the other services.
```

Comment:
This section seems like it's targeted at developers. If so, should note that explicitly.